tests/unit/plugins/modules/kubevirt_fixtures.py

```diff
@@ -2,15 +2,14 @@
 __metaclass__ = type
 
 import pytest
+import openshift.dynamic
```

Comment:
This is a bit strange. I thought collection does not use openshift client anymore?

Reply:
@tadeboro its a good point, I hit lots of errors when moving to kubernetes.dynamic so thought I'd break down into two changesets but am re-attempting to debug the original errors now.